Source/CarthageKit/Project.swift

```diff
@@ -684,50 +698,85 @@ public final class Project {
 			.then(SignalProducer<(), CarthageError>.empty)
 	}
 
+	/// - parameters:
+	/// 	- flatMapTransform: defaults to identity function. Only ever used for one thing, but parameterized for unit testing purposes.
```

Comment:
I have no idea what it's transforming. Nor do I know the values that this function sends.